bot/src/main/java/com/community/tools/model/TraineeMentorRelationId.java

```diff
@@ -0,0 +1,15 @@
+package com.community.tools.model;
+
+import java.io.Serializable;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@EqualsAndHashCode
+@NoArgsConstructor
+public class TraineeMentorRelationId implements Serializable {
```

Comment:
It seems like you was trying to normalize the DB structure. But in this case would be better just add a mentor ID into a trainee entity